src/components/dashboard/settingsLayout/SettingsLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import { Heading } from "../heading/Heading";
+import { Tabs } from "../tabs/Tabs";
+
+import type { ReactNode } from "react";
+
+const tabs = [
+  {
+    label: "General",
+    pathname: "/dashboard/settings",
+  },
+];
```

Comment:
I will make `/settings` folder, move all components related to it there (without `Settings` prefix) and move something like this to consts file in `/config`